openedx/core/djangoapps/lang_pref/api.py

```diff
@@ -14,6 +16,20 @@
 Language = namedtuple('Language', 'code name')
 
 
+def header_language_selector_is_enabled():
+    """Return true if the header language selector has been enabled via settings or site-specific configuration."""
+    setting = get_value('SHOW_HEADER_LANGUAGE_SELECTOR', settings.FEATURES.get('SHOW_HEADER_LANGUAGE_SELECTOR', False))
+
+    # The SHOW_LANGUAGE_SELECTOR setting is deprecated, but might still be in use on some installations.
+    deprecated_setting = get_value('SHOW_LANGUAGE_SELECTOR', settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False))
+    return setting or deprecated_setting
```

Comment:
The only other thing is to maybe log a warning "honoring deprecated setting SHOW_LANGUAGE_SELECTOR see X for new settings" but I otherwise like the backcompat work.